.vscode/settings.json

```diff
@@ -12,5 +12,6 @@
   "[ruby]": {
     "editor.defaultFormatter": "Shopify.ruby-lsp",
     "editor.formatOnSave": true
-  }
+  },
+  "sorbet.highlightUntyped": "everywhere-but-tests"
```

Comment:
I think this won't work correctly for this repo?

> Sorbet treats any file in a folder named test or ending with the .test.rb suffix as a test file. This is not yet configurable

Reply:
Good point, I'll open a PR to include `spec/` and `_spec`. In the meantime I don't think this is a blocker for the PR.

Reply:
https://github.com/sorbet/sorbet/pull/7730